packages/suite-desktop/src/modules/coinjoin.ts

```diff
@@ -51,11 +55,16 @@ export const init: Module = ({ mainWindow }) => {
                     if (method === 'enable') {
                         logger.debug(SERVICE_NAME, `CoinjoinClient enable ${params}`);
                         const response = await client.enable();
-                        // TODO: start coinjoin middleware binary
-                        return response;
+                        try {
```

Comment:
i cannot comment on unchanged lines.

lets generate free port on line 50, override `settings.middlewareUrl` to localhost:freeport and use those settings in constructor:
```
settings.middlewareUrl = 'http://localhost:{port}/Cryptography'
new CoinjoinClient(settings);
```

Reply:
Done